projects/bp-gallery/src/views/picture/comments/FormattedComment.tsx

```diff
@@ -36,4 +47,42 @@ const FormattedComment = ({ comment }: { comment: FlatComment }) => {
   );
 };
 
+const CommentEditField = ({ comment }: { comment: FlatComment }) => {
+  const commentRef = useRef<FlatComment>(comment);
+
+  const [updateComment] = useFixCommentTextMutation({
+    refetchQueries: ['getPictureInfo'],
+  });
+
+  useEffect(() => {
+    commentRef.current = comment;
+  }, [comment]);
+
+  const config = {
+    ...defaultJoditConfig,
+    readonly: false,
+  };
+
+  const onBlur = useCallback(() => {
+    // Save description
```

Comment:
```suggestion
    // Save comment
```

Or even remove the whole comment :D